tests/test_encoder.py

```diff
@@ -128,6 +128,34 @@ def test_encode(self):
 END;'''
         self.assertEqual(s, self.e.encode(m))
 
+    def test_encode_quantity(self):
+        p = (Units(34, 'm/s'), '34 <m/s>')
+        self.assertEqual(p[1], self.e.encode_quantity(p[0]))
```

Comment:
[picky] i'm not a fan of this style tbh. if you are trying to keep the line length short one could do:
```
q, e = Units(34, 'm/s'), '34 <m/s>'
self.assertEqual(e, self.e.encode_quantity(q))
```
or just directly
```
self.assertEqual('34 <m/s>', self.e.encode_quantity(Units(34, 'm/s')))
```

again this is mainly a style clash, the code should work as is

Reply:
i'd agree that the `q, e` unpacking is easier to read, guiding the mind what is happening, and therefore the assert line is easier to understand.

Reply:
I'm happy to change this one (although I'd use different letters, just because 'e' and 'self.e' are really too close for my comfort.  It was built like this because I have other tests structured this way,  It allows me to have a whole list of 'p' values (pairs of truth and test), and then run the list.  However, there is only one of these, and I take the point about unpacking.  I'll push a commit.

Did you even test the units stuff, which is the real meat?  I will take it as a good thing that you're nit picking about the style on the tests, must mean the other stuff is good. :)

Reply:
i'm still digging through the docs, you write so much.. ;)

Reply:
Yay, will take as compliment!